requirements.txt

```diff
@@ -2,20 +2,20 @@ Jinja2==2.7.3
 Mako==1.0.0
 MarkupSafe==0.23
 Pygments==2.0.1
-SQLAlchemy==0.9.8
-alembic==0.7.1
+SQLAlchemy==1.0.5
+alembic==0.7.6
 argparse==1.2.1
 backports.ssl-match-hostname==3.4.0.2
 certifi==14.05.14
 fancycompleter==0.4
--e git+git@github.com:quantopian/ipython.git@8fd1a6f42bd2785c902ab0411b50afa4051699b1#egg=ipython-master
+ipython==3.1.0
```

Comment:
@richafrank any reason not to bump us to 3.2 here?  I don't think there are any changes that would break our internal deployment (though we should test that to be sure).